.eslintrc.yaml

```diff
@@ -199,7 +199,7 @@ rules:
   newline-per-chained-call: [0]
   no-alert: [0]
   no-array-constructor: [2]
-  no-async-promise-executor: [2]
+  no-async-promise-executor: [0]
```

Comment:
Note, this rule is actually bullshit, I've long disabled it in my personal config :)